autopilot/autopilot.go

```diff
@@ -574,12 +574,6 @@ func (ap *Autopilot) performWalletMaintenance(ctx context.Context) error {
 	b := ap.bus
 	l := ap.logger
 
-	// no contracts - nothing to do
```

Comment:
Fine to revert but I'd remove this check. We used to calculate the amount of outputs we wanted in relation to `cfg.Contracts.Amount`. If the user hasn't yet configured the amount of contracts he wants, there's no reason not to prepare the wallet so he can hit the ground running when he does configure it.